util/libcrypto.num

```diff
@@ -5163,18 +5163,6 @@ ERR_load_OSSL_DECODER_strings           ?	3_0_0	EXIST::FUNCTION:
 OSSL_DECODER_gettable_params            ?	3_0_0	EXIST::FUNCTION:
 OSSL_DECODER_get_params                 ?	3_0_0	EXIST::FUNCTION:
 OSSL_DECODER_CTX_new_by_EVP_PKEY        ?	3_0_0	EXIST::FUNCTION:
-EVP_PKEY_set1_X25519                    ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get0_X25519                    ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get1_X25519                    ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_set1_X448                      ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get0_X448                      ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get1_X448                      ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_set1_ED25519                   ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get0_ED25519                   ?	3_0_0	EXIST::FUNCTION:EC
-EVP_PKEY_get1_ED25519                   ?	3_0_0	EXIST::FUNCTION:EC
```

Comment:
Will deleting these cause issues?

Reply:
Well, er, no.  The whole idea with this PR is exactly this, to unpublish what was published by mistake

Reply:
Or, it *shouldn't* be an issue.  If there is, I need to tinker this PR a little more

Reply:
I just checked all the builds, and they don't indicate any problem.  So...